### DISCUSSION TITLE
Requesting Clarification on ifc2ca's Functionality and Integration with IFC

### DISCUSSION THREAD

**Discussion by kulasegaram** [IfcOpenShell/IfcOpenShell#4356] [Q&A]
Dear @Jesusbill,  
  
I hope this message finds you well. I have recently come across the projects "ifc2ca" and "ADA", both of which share a focus on IFC and structural analysis, similar to the project "IfcTruss". With the intention of showcasing similar projects on the documentation page of IfcTruss, I would like to request some clarification about ifc2ca's functionality and integration with IFC.  
  
Firstly, could you kindly confirm that the line, shell entities, and load entities from IfcStructuralAnalysisDomain are utilized in ifc2ca for structural analysis in Code_Aster? Furthermore, does ifc2ca support solid elements, and does it  
solely focuses on Code_Aster?  
  
Furthermore, I would appreciate your confirmation that the results from the FE analysis performed using ifc2ca and Code_Aster are not saved back into the IFC file.  
  
Regarding the installation, could you please confirm whether Code_Aster can be installed via Pip or Conda? Additionally, is it compatible with all major operating systems?  
  
[IfcTruss](https://github.com/IfcTruss/IfcTruss) also employs entities from IfcStructuralAnalysisDomain to represent exclusively truss structures and point loads for structural analysis. The results are then saved back into the IFC file. All dependencies and IfcTruss itself can be installed using "pip install ifctruss" on all three major operating systems.  
  
Your insights and clarification on these points would be greatly appreciated, as they will assist me in creating an accurate comparison on the [documentation page of IfcTruss](https://ifctruss.github.io/IfcTruss/). Thank you for your time and assistance.

**Comment by Moult** [comment]
BTW added IfcTruss to https://wiki.osarch.org/index.php?title=AEC_Free_Software_directory#Structural.2C_thermal_and_computational_fluid_dynamics_.28CFD.29_analysis
